server/src/test/java/org/apache/cassandra/sidecar/cdc/CdcConfigImplTest.java

```diff
@@ -150,103 +137,11 @@ void testConfigsWhenConfigsAreNotEmpty()
         assertThat(cdcConfig.persistDelay()).isEqualTo(new MillisecondBoundConfiguration(5, TimeUnit.SECONDS));
     }
 
-    @Test
-    void testConfigChanged() throws Exception
```

Comment:
Why did you remove this test? I think this is valuable to have a test for checking callback and config updating when config is changed.

Reply:
This became a periodic task, and the callback was moved to a vertx signal instead.